### PR TITLE
Readme

### PR DESCRIPTION
**Pull request by Erykahh1982**
Sziasztok,

Please check what I have created so far for our Project as a READM file, thank you!
Erika

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,3 +1,48 @@
...
-This is the repository of Tribes Front-End Project of Team Godess
+# Project Summary
+
+Tribes by GFA-Goddess, this is the front-end part of a strategy game developed by our team as a coursework in Green Fox Academy
+
+This project was generated with Angular CLI version 7.3.9. | Angular 7.2.15. | Node 10.15.3
+
+# Getting Started
+
+$ npm install -g @angular/cli through command line and download VS Code to   your PC from code.visualstudio.com
```
Comment (balazsbarni) [inline README.md]: VS code not needed, anyone can use his/hers favorite text editor

File: README.md [hunk 2]
```diff
@@ -1,3 +1,48 @@
...
+
+Install show-hide password module / follow the instructions in the link below:
```
Comment (balazsbarni) [inline README.md]: I think all the Install happens if you cloned the repo, and type `npm install`

File: Tribes/README.md [hunk 3]
```diff
@@ -1,10 +1,29 @@
-# Tribes
+# Project Summary
```
Comment (balazsbarni) [inline Tribes/README.md]: this is not needed I think

**Comment by Erykahh1982** [comment]
Hi Barni,

I have modified the file content a bit, please check if this is the expected outcome.
Thank you
Erika

File: README.md [hunk 4]
```diff
@@ -1,3 +1,50 @@
 # godess-tribes-frontend
 
 This is the repository of Tribes Front-End Project of Team Godess
+
+# Project Summary
```
Comment (EggDice) [inline README.md]: Please use a hierarchy for headings, one page should consist of one top-level heading.

File: README.md (hunk 2 above)
Comment (EggDice) [reply README.md]: Yes please mention `npm install`

**Pull request merged**